src/bumper/bumper.go

```diff
@@ -10,6 +11,8 @@ import (
 	"github.com/newrelic/release-toolkit/src/version"
 )
 
+var ErrNoNewVersion = errors.New("no newer version to be bumped")
```

Comment:
I'm still not convinced about the erroring logic being in the bumper. I think the action we want to take is fundamentally a user-facing one, so I think it may be better for this logic to be in the command instead.

Reply:
But if we do that, we will need the command to iterate all the tags to get the first. The command doesn't know nothing about the current version.

Reply:
And I see it here, I expect the BumpSource command to Bump the source or tell me it couldn't.

Reply:
> But if we do that, we will need the command to iterate all the tags to get the last. The command doesn't know nothing about the current version.

When I wrote this I had, for some reason, the wrong idea that the command was dealing with `bump.Type` directly. This is in fact not true, so your comment is on point. `bump.Bump` should not return an error (just `bump.None` if there is none), but `bumper.Bump` should return an error so the command can catch it.